src/processor/Package.ts

```diff
@@ -69,6 +70,17 @@ export class Package {
     } else if (resource instanceof ExportableMapping) {
       checkDuplicateDefinition(this.mappings, resource, 'mapping');
       this.mappings.push(resource);
+    } else if (resource instanceof ExportableAlias) {
+      const duplicateAlias = this.aliases.find(e => (e.alias = resource.alias));
```

Comment:
We're gonna want a comparison operator here instead of the singular `=`. As it is right now, this always results in duplicates being flagged when there is more than one alias in the file.
```suggestion
      const duplicateAlias = this.aliases.find(e => (e.alias === resource.alias));
```